person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ ] I would like to recommend you make sure that all your instance variables have `attr_accessor` since there are both getter and setter.  attr_reader defines only getter.  Please fix it to make your code work as expected :+1:

Reply:
The code has been refactored to use the `attr_accessor`.